variables.tf

```diff
@@ -98,3 +98,7 @@ variable "enabled_metrics" {
 variable "subnet_ids" {
   type = "list"
 }
+
+variable "ecs_service_role_name" {
```

Comment:
Can you please move this up under `ecs_for_ec2_service_role_name` in `variables.tf`?

Reply:
Hi @hectcastro thank you for the prompt response. `ecs_service_role_name` has been moved up as per your suggestion.